packer/build.go

```diff
@@ -79,6 +84,14 @@ func (b *CoreBuild) GetPluginsMetadata() map[string]PluginDetails {
 	return resp
 }
 
+func (b *CoreBuild) GetMetadata() BuildMetadata {
+	metadata := BuildMetadata{
+		PackerVersion: version.FormattedVersion(),
+		Plugins:       b.GetPluginsMetadata(),
```

Comment:
Since GetPluginsMetadata is only used within this method let’s not export it incase we need to change the implementation later on. 

This might be me drawing a line between HCP Packer and Packer but I do have some reservation about having metadata on CoreBuild. 

But it is a good starting point to go with for now until we know more.

Reply:
Done